gatherers/theme-color.js

```diff
@@ -20,15 +20,15 @@ const Gather = require('./gather');
 
 class ThemeColor extends Gather {
 
-  static gather(options) {
+  afterPageLoad(options) {
     const driver = options.driver;
 
     return driver.querySelector('head meta[name="theme-color"]')
       .then(node => node && node.getAttribute('content'))
       .then(themeColorMeta => {
-        return {themeColorMeta};
+        this.artifact = {themeColorMeta};
       });
   }
 }
 
-module.exports = ThemeColor;
+module.exports = new ThemeColor();
```

Comment:
plus it makes it really easy to unknowingly hang state on these exports which is then shared by everyone.